pagespeed/automatic/Makefile

```diff
@@ -24,7 +24,7 @@
 #
 # When running this Makefile from the 'automatic' directory then it will
 # be set automatically.
-MOD_PAGESPEED_ROOT = $(shell cd ../../..; pwd)
+MOD_PAGESPEED_ROOT = $(shell cd ../..; pwd)
```

Comment:
Something is still needed, but I think not quite this.  Paths are slightly different inside Google and I'm pretty sure we need ../../.. outside Google and ../.. inside.  I'll fix this in a separate change.